src/mame/drivers/uknc.cpp

```diff
@@ -9,97 +9,946 @@ Graphics 640x288 pixels.
 
 2009-05-12 Skeleton driver.
 
-Status: both CPUs start in the weeds.
+Status: snapshot of work in progress.  needs real VM2 CPU core.
 
 ****************************************************************************/
 
 #include "emu.h"
+
+#include "bus/qbus/qbus.h"
 #include "cpu/t11/t11.h"
+#include "imagedev/cassette.h"
+#include "machine/bankdev.h"
+#include "machine/timer.h"
+
 #include "emupal.h"
 #include "screen.h"
 
 
+#define UKNC_TOTAL_HORZ (800)
```

Comment:
We prefer "static constexpr" for constants like this over preprocessor macros now.

Reply:
changed.